doc/source/toolchain.rst

```diff
@@ -283,3 +293,5 @@ References
 .. [8] https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B#Internal_version_numbering
 .. [9] https://wiki.python.org/moin/WindowsCompilers
 .. [10] https://numpy.org/neps/nep-0029-deprecation_policy.html
+.. [11] https://devblogs.microsoft.com/cppblog/c11-and-c17-standard-support-arriving-in-msvc/
+.. [12] https://en.wikipedia.org/wiki/C11_(C_standard_revision)#Optional_features
```

Comment:
Unfortunately, sphinx wrongly breaks this link at the `#` - how can I indicate that this shouldn't be broken? Backticks?

Reply:
try replacing left and right brackets with `%28` and `%29`

Reply:
Thanks for the tip! :)